src/ray/core_worker/core_worker.cc

```diff
@@ -516,11 +526,30 @@ Status CoreWorker::Create(const std::shared_ptr<Buffer> &metadata, const size_t
 
 Status CoreWorker::Create(const std::shared_ptr<Buffer> &metadata, const size_t data_size,
                           const ObjectID &object_id, std::shared_ptr<Buffer> *data) {
-  return plasma_store_provider_->Create(metadata, data_size, object_id, data);
+  if (!local_mode_enabled_) {
+    return plasma_store_provider_->Create(metadata, data_size, object_id, data);
+  } else {
+    return Status::NotImplemented(
+        "Create without list...pre-existing ObjID not supported");
+  }
 }
 
 Status CoreWorker::Seal(const ObjectID &object_id, bool pin_object,
-                        const absl::optional<rpc::Address> &owner_address) {
+                        const absl::optional<rpc::Address> &owner_address,
+                        const absl::optional<std::shared_ptr<Buffer>> &data,
+                        const absl::optional<std::shared_ptr<Buffer>> &metadata) {
+  if (local_mode_enabled_) {
```

Comment:
Looks like we're basically just duplicating `Put()` here. Could we just use the existing `Put()` interface for local_mode instead?

Reply:
+1, Currently it looks strange that `Seal` also needs to take `data` and `metadata`.